### DISCUSSION TITLE
Does canvas blocker have font fingerprint defence?

### DISCUSSION THREAD

**Discussion by maestropss** [kkapsner/CanvasBlocker#583] [General]
Hello. Thank you for your add-on. 
The question is if canvas blocker has font fingerprint defence. Or should we use Font Fingerprint Defender add-on as an extra defence? Thanks

**Comment by kkapsner** [comment]
No, CB does not have font fingerprint defence, but you should not use Font Fingerprint Defender as it does not properly protect against it and can be detected very easily (last time I checked).
Check #200 for more details about the topic.
